exp/api/v1alpha3/types.go

```diff
@@ -154,3 +155,53 @@ func LaunchTemplateNeedsUpdate(incoming *AWSLaunchTemplate, existing *AWSLaunchT
 
 	return false
 }
+
+// ASGState contains the state of the ASG. e.g. pending, running, etc
+type ASGState string
+
+//TODO: maybe make it match this more? https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroupLifecycle.html
+var (
```

Comment:
I don't know how many of these are actually applicable?

Reply:
Yeah I was wondering if these were things I had just initially missed from the conditions PR. Will remove. 
Because this would be more applicable to actual EC2 instances rather than the ASG itself?